addons/uh60_fms/config/interaction.hpp

```diff
@@ -190,6 +190,12 @@ class hmd {
   FMS_BTN(FMS_3,"METRIC DCLT","")
     buttonUp = "[4] call vtx_uh60_anvishud_fnc_toggleHud;";
   };
+  FMS_BTN(FMS_4,"BRT","")
+    buttonUp = [(_this select 0), ((driver vehicle player) isEqualTo player), 1] call vtx_uh60_anvishud_fnc_changeBrightness;
```

Comment:
Seems like you don't need to pass the 2nd parameter, since the evaluation is always the same?

Reply:
This is how I determine if the player calling the function is pilot or copilot. In local testing it was giving me true for right seat and false for right seat, even after taking controls. 

I had set it up this way based on the feedback from #217.

Reply:
right, but `player` is already unique where it's run, and I don't think we intend to change the other player's brightness. So it's not needed as a parameter and can just be evaluated inside `changeBrightness`.

Reply:
Gotcha, makes sense. Addressed in this commit:
https://github.com/Project-Hatchet/H-60/pull/247/commits/6c7f45809cd07a2f7eaa0fbf14eaaebc7414dd33